personal_graph/__init__.py

```diff
@@ -1,3 +1,15 @@
+from .graph import Graph
```

Comment:
easier to just do:
```py
from personal_graph.graph import *
```

while * imports are discouraged elsewhere, they make sense in `__init__.py` files

Reply:
Alright

Reply:
using * gives mypy error, we can use `from personal_graph.ml import to_networkx, from_networkx` instead.